workflow/documentation.md

```diff
@@ -0,0 +1,172 @@
+# BINDING-SCANNER: workflow documentation
+
+This document describes the individual steps of the workflow. For instructions
+on installation and usage please see [here](../README.md).
+
+## Table of Contents
+- [**Description of workflow steps**](#description-of-workflow-steps)
+  - [**Rule graph**](#rule-graph)
+  - [**Preparatory**](#preparatory)
+    - [**Config file**](#config-file)
+  - [**Snakemake Rules**](#snakemake-rules)
+    - [**all**](#all)
+    - [**create_results_directory**](#create_results_directory)
+    - [**plot_sequence_logos**](#plot_sequence_logos)
+    - [**prepare_MotEvo_parameters**](#prepare_MotEvo_parameters)
+    - [**prepare_sequence_for_MotEvo**](#prepare_sequence_for_MotEvo)
+    - [**run_MotEvo_analysis**](#run_MotEvo_analysis)
+    - [**combine_MotEvo_results**](#combine_MotEvo_results)
+    - [**plot_heatmap_of_MotEvo_results**](#plot_heatmap_of_MotEvo_results)
+
+## Description of workflow steps
+
+> The workflow consists of one Snakemake file:
+> It contains general steps for the creation of indices and
+> other required files derived from the annotations, steps that are applicable
+> to both sequencing modes, and steps that deal with gathering, summarizing or
+> combining results. Individual steps of the workflow are described briefly. 
```

Comment:
We do not do that here, these are RNA-Seq data processing steps, *zarp* pipeline does them.